themes/vex-hugo/layouts/partials/promo.html

```diff
@@ -6,7 +6,7 @@
     <div class="container">
         <div class="row">
             <div class="col-md-6 text-center mb-5 mb-lg-0">
-                <img class="img-fluid" src="{{ .image | absURL }}" alt="">
+                <img class="img-fluid" src="{{ .image | absURL }}" height="177" width="178" alt="">
```

Comment:
An alt text (e.g. as parameter) would be great here.

Reply:
done